src/index.html

```diff
@@ -1,15 +1,155 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>HTML Form</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>HTML Form</h1>
-  <script type="text/javascript" src="./main.js"></script>
-</body>
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML Form</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="POST">
+      <fieldset>
+        <legend>Personal information</legend>
+        <div class="block">
+          <label for="Surname">Surname:</label>
+          <input
+          type="text"
+          name="surname"
+          id="Surname"
+          required
+          autocomplete="off">
+        </div>
+        <div class="block">
+          <label for="Name">Name:</label>
+          <input
+          type="text"
+          name="name"
+          id="Name"
+          required
+          autocomplete="off">
+        </div>
+        <div class="block">
+          <label for="Age">How old are you?</label>
+          <input
+          type="number"
+          name="age"
+          id="Age"
+          min="12"
+          max="100"
+          value="12">
+        </div>
+        <div class="block">
+          <label for="Date">Full date of birth:</label>
+          <input
+          type="date"
+          name="date"
+          id="Date"
+          required>
+        </div>
+        <div class="block">
+          <label for="Acception">I accept the term of the agreement:</label>
+          <input
+          type="checkbox"
+          name="acception"
+          id="Acception"
+          required>
+        </div>
```

Comment:
```suggestion
        <div class="block">
          <label for="Acception">
            I accept the term of the agreement:
            <input
              type="checkbox"
              name="acception"
              id="Acception"
              required
            >
          </label>
        </div>
```
отредактируй в соответсвии с этим примером